pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -34,16 +36,15 @@
     "System.Runtime.Loader": "4.3.0",
     "System.Security.Cryptography.Algorithms": "4.3.1",
     "System.Security.Cryptography.Encoding": "4.3.0",
+    "System.Security.Cryptography.OpenSsl": "4.3.1",
     "System.Security.Cryptography.Primitives": "4.3.0",
     "System.Security.Cryptography.X509Certificates": "4.3.2",
-    "System.Security.Cryptography.OpenSsl": "4.3.1",
     "System.Threading.Tasks.Dataflow": "4.7.0",
     "System.Threading.Tasks.Extensions": "4.3.0",
     "System.Threading.Tasks.Parallel": "4.3.0",
     "System.Threading.Thread": "4.3.0",
     "System.Threading.ThreadPool": "4.3.0",
-    "Microsoft.DiaSymReader.Native": "1.4.1",
-    "Libuv": "1.9.2",
+    "System.Xml.ReaderWriter":"4.3.1-servicing-26001-01",
     "runtime.native.System.IO.Compression": "4.3.1",
```

Comment:
r < s :)

Reply:
hahah you're right, missed to move those up :)